scripts/linters/custom_eslint_checks/rules/no-testonly.js

```diff
@@ -32,15 +32,22 @@ module.exports = {
     fixable: null,
     schema: [],
     messages: {
-      notTestOnlyAllowed: 'Can not call a function containing \'testOnly\' ' +
-        'in a JS/TS file that is not a .spec file'
+      notTestOnlyAllowed: 'Please do not call a testOnly function from a ' +
+        'non-test file.'
     }
   },
   create: function(context) {
+    var validateCallExpression = function(node) {
+      if (typeof node.callee.name === 'string' &&
```

Comment:
I don't think we need to check if callee.name is a string, it will always going to be a string right?

Reply:
Not all node.callee had the property _name_.  And I was getting an error because of the undefined value. 

To improve this I reviewed the structure of the **node** object and I saw that the property _type_ always exists. And if **node.callee.type === 'Identifier'** the called function is referenced, and in this case the property _name_  always exists.